src/uniprotkb/components/protein-data-views/NightingaleZoomTool.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from 'react';
+import { SearchIcon } from 'franklin-sites';
+import useCustomElement from '../../../shared/hooks/useCustomElement';
+
+import './styles/nightingale-zoom-tool.scss';
+
+// Icons and icon size TBD once designed.
+const iconSize = 24;
+
+const NightingaleZoomTool: FC<{ length: number }> = ({ length }) => {
+  useCustomElement(
+    /* istanbul ignore next */
+    () =>
+      import(
+        /* webpackChunkName: "protvista-zoom-tool" */ 'protvista-zoom-tool'
+      ),
+    'protvista-zoom-tool'
+  );
+
+  return (
+    <div className="button-group">
+      <protvista-zoom-tool length={length}>
+        <span slot="zoom-in" className="nightingale-button-content">
+          <SearchIcon width={iconSize} />+
```

Comment:
Not a fan of relying on overlapping text and SVG. This will change depending on user fonts, browsers, or even screens (mine show the "-" not centred, towards the left, and same thing for the "+" but towards the right)
The "+", "-", and "ATG" (or whatever is chosen for this) need to be SVG shapes within the same SVG.